UDV-Core/src/Utils/3DTiles/3DTilesUtils.js

```diff
@@ -167,6 +167,191 @@ export function setTileVerticesColor(tile, newColor, indexArray = null) {
   }
 }
 
+/**
+ * Creates tile groups and associate them with the given materials.
+ * 
+ * @param {*} tile The 3DTiles tile.
+ * @param {Array<any>} materials An array of material parameters. Each entry
+ * should be an object that will be passed as parameter in the THREE.Material
+ * constructor. Accepted values are for example `color` or `opacity` (see
+ * THREE.js documentation).
+ * @param {Array<any>} ranges An array of ranges. A range is an object with 3
+ * propeties :
+ * - `start`: the start index of the group of vertices
+ * - `count`: the number of vertices of the group
+ * - `material`: the index of the material in the materials array
+ */
+export function createTileGroups(tile, materials, ranges) {
```

Comment:
Could you rename materials to something like materialsProperties please to avoid confusion with actual threejs materials and with the materials of the mesh ?

Could you also add a list of the accepted properties please ?

Reply:
I renamed it `materialsProps` to keep a short name.

I didn't add the full list of accepted properties because it would be too long. The purpose of this object is to be passed as parameter for the `MeshLambertMaterial` constructor. According to the doc :

> Any property of the material (including any property inherited from Material) can be passed in here.

Instead, I added a comment to explain the above and a link to the relevant documentation (https://threejs.org/docs/#api/en/materials/MeshLambertMaterial).

Reply:
Perfect, thanks